app/customer.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from app.car import Car
+
+
+class Customer:
+    def __init__(
+        self, name: str, products: dict, location: list, money: int, car: Car
+    ) -> None:
+        self._name = name
+        self._products = products
+        self._location = location
+        self._money = money
+        self._car = car
+
+    @classmethod
+    def create_customer(cls, customer_info: dict) -> Customer:
+        try:
+            return cls(
+                customer_info["name"],
+                customer_info["product_cart"],
+                customer_info["location"],
+                customer_info["money"],
+                Car.create_car(customer_info["car"]),
+            )
+        except KeyError as e:
+            print(e, "not exist")

```

Comment:
Similar to the `Car` class, the error handling in the `create_customer` method catches `KeyError`, but the error message 'not exist' is not very informative. Consider providing more context in the error message, such as which key is missing, to make debugging easier.